sensors/gcp_message_sensor.py

```diff
@@ -125,3 +137,22 @@ def _try_deserialize(body):
             pass
 
         return body
+
+    def file_parameter(self, contents, file_name):
```

Comment:
same with this method

Reply:
Also, we have this same method in 3 places, it should go into a util file

Reply:
I'll try.

Reply:
I could create the util file for the sensors but it can not used in the action.

Reply:
I see